[2023-04-28T01:09:08.512+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gsc_to_bq.create_bucket_raw scheduled__2023-04-27T00:00:00+00:00 [queued]>
[2023-04-28T01:09:08.521+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: gsc_to_bq.create_bucket_raw scheduled__2023-04-27T00:00:00+00:00 [queued]>
[2023-04-28T01:09:08.522+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2023-04-28T01:09:08.523+0000] {taskinstance.py:1280} INFO - Starting attempt 2 of 2
[2023-04-28T01:09:08.524+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2023-04-28T01:09:08.539+0000] {taskinstance.py:1300} INFO - Executing <Task(GCSCreateBucketOperator): create_bucket_raw> on 2023-04-27 00:00:00+00:00
[2023-04-28T01:09:08.544+0000] {standard_task_runner.py:55} INFO - Started process 15704 to run task
[2023-04-28T01:09:08.548+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'gsc_to_bq', 'create_bucket_raw', 'scheduled__2023-04-27T00:00:00+00:00', '--job-id', '436', '--raw', '--subdir', 'DAGS_FOLDER/gcs_to_bq.py', '--cfg-path', '/tmp/tmpl5_dkvnf']
[2023-04-28T01:09:08.551+0000] {standard_task_runner.py:83} INFO - Job 436: Subtask create_bucket_raw
[2023-04-28T01:09:08.729+0000] {task_command.py:388} INFO - Running <TaskInstance: gsc_to_bq.create_bucket_raw scheduled__2023-04-27T00:00:00+00:00 [running]> on host b592efbffeab
[2023-04-28T01:09:08.807+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Junior Lafuente
AIRFLOW_CTX_DAG_ID=gsc_to_bq
AIRFLOW_CTX_TASK_ID=create_bucket_raw
AIRFLOW_CTX_EXECUTION_DATE=2023-04-27T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-04-27T00:00:00+00:00
[2023-04-28T01:09:08.819+0000] {base.py:73} INFO - Using connection ID 'gcp' for task execution.
[2023-04-28T01:09:08.843+0000] {gcs.py:880} INFO - Creating Bucket: bucket_raw_jr; Location: us-central1; Storage Class: STANDARD
[2023-04-28T09:16:39.495+0000] {local_task_job.py:273} WARNING - State of this instance has been externally set to failed. Terminating instance.
[2023-04-28T09:16:39.525+0000] {process_utils.py:133} INFO - Sending Signals.SIGTERM to group 15704. PIDs of all processes in the group: [15704]
[2023-04-28T09:16:39.527+0000] {process_utils.py:84} INFO - Sending the signal Signals.SIGTERM to group 15704
[2023-04-28T09:16:39.534+0000] {taskinstance.py:1479} ERROR - Received SIGTERM. Terminating subprocesses.
[2023-04-28T09:16:40.115+0000] {process_utils.py:79} INFO - Process psutil.Process(pid=15704, status='terminated', exitcode=0, started='01:09:07') (15704) terminated with exit code 0
